tests/operators/test_python.py

```diff
@@ -1155,6 +1169,21 @@ def f():
         )
         copy.deepcopy(task)
 
+    def test_except_value_error(self):
+        def f():
+            return 1
+
+        task = PythonVirtualenvOperator(
+            python_callable=f,
+            task_id="task",
+            dag=self.dag,
+        )
+
+        task.log.error = unittest.mock.Mock()
```

Comment:
Maybe this should use `caplog` to actually capture the logs instead of mocking it.

Reply:
I replaced `log.error` with a custom exception class, see below